### PR TITLE
success feature comment 

### PR DESCRIPTION
**Pull request by aofattaporn**
- create grobal state of comment bloc 
- refactor class state Comment 
- refactor class bloc Comment 
- refactor class eveent Comment 

**Adding new file about Constant **
- constant URL 
- constant font

### CODE REVIEW

**Comment by aofattaporn** [comment]
@teamchz  @thunyathep  @sura1542  @Balloii

**Review by thunyathep — APPROVED** [review]

File: lib/src/screens/post/ViewPost.dart [hunk 1]
```diff
@@ -46,547 +59,530 @@ class ViewPostState extends State<ViewPost> {
...
+                    return const CupertinoAlertDialog(
+                      title: Text("Deleting Your Comment..."),
+                      // CircularProgressIndicator(color: thirterydColor)),
+                    );
+                  });
+            } else if (state is DeletedComment) {
+              showToast("delete comment success");
+              Navigator.of(context).pop();
+              Navigator.of(context).pop();
+              Navigator.of(context).pop();
```
Comment (teamchz) [inline lib/src/screens/post/ViewPost.dart:79]: Curious why there are too many Navigator.pop?

**Review by teamchz — COMMENTED** [review]
A little question about ViewPost.dart

File: lib/src/screens/post/ViewPost.dart (hunk 1 above)
Comment (aofattaporn) [reply lib/src/screens/post/ViewPost.dart:79]: 5555555555555

 this is just a stupid solution for pop from **delete pop-up** to **page View All Post**  when delete success 
 
 I don't know what I need to do about this problem, you can switch to this branch and find a good solution for this! 🥲🙏

Comment (aofattaporn) [reply lib/src/screens/post/ViewPost.dart:79]: but you do not have time for this problem, I will goo solution more in tomorrow, or less than

**Review by teamchz — APPROVED** [review]

**Pull request merged**